tests/__fixtures__/Rules/newline_before_scenario/fixture.js

```diff
@@ -0,0 +1,559 @@
+const { format } = require("util");
+const generator = require("../../../helpers/problemGenerator");
+const NewlineBeforeScenario = require("../../../../lib/rules/newline_before_scenario");
+
+function generateProblem(location, actual, expected = 1) {
+    return generator(
+        NewlineBeforeScenario,
+        location,
+        format(NewlineBeforeScenario.meta.message, expected, actual)
+    );
+}
+
+function getValidTestData() {
+    return [
+        [
+            "with Rule: Scenario",
+            `Feature: a feature file
+  Rule: a rule
+
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: Background - Scenario",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: Scenario Outline",
+            `Feature: a feature file
+  Rule: a rule
+
+    Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "with Rule: Background - Scenario Outline",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+
+    Scenario Outline: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: multiple scenarios",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+
+    Scenario: a scenario
+
+    Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "with Rule: description - Scenario",
+            `Feature: a feature file
+  Some feature description
+  is here
+  Rule: a rule
+    rule description here
+
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: Scenario tags",
+            `Feature: a feature file
+  Rule: a rule
+
+    @tag1 @tag2
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: Scenario multiline tags",
+            `Feature: a feature file
+  Rule: a rule
+
+    @tag1
+    @tag2
+    Scenario Outline: a scenario outline`,
+            [],
+        ],
+        [
+            "with Rule: Scenario comments",
+            `Feature: a feature file
+  Rule: a rule
+
+    #comment line
+    Scenario: a scenario`,
+            [],
+        ],
+        [
+            "with Rule: description - Scenario tags and comments",
+            `Feature: a feature file
+  feture description here
```

Comment:
```suggestion
  feature description here
```

The typo does not matter, but you may as well fix it - occurs in about 8 places.